### ISSUE TITLE
[T2][Accessibility][Functional] - Unable to move from the "Past Medical" or any tab using keyboard.

### ISSUE THREAD

**Issue by Tanushekhawat** [Tanushekhawat/Tanu#7]
Steps to reproduce:

Unable to move from the "Past Medical" or any tab using keyboard.
